.github/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+**Description**: Describe your changes.
+
+**Related issue:** Add link to related issue.
+
+**Documentation changes required:** (Y/N) Mark if documentation changes are required.
```

Comment:
I think it is better to create check boxes instead of `(Y/N)`: `- [ ]`

Reply:
Done, thank you!